src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public String toString() {
```

Comment:
When overriding the toString method from the Object class, it's a good practice to use the @Override annotation. This helps to make the intention clear and can prevent issues if the signature of the overridden method changes.